docs/reference/networking.md

```diff
@@ -0,0 +1,17 @@
+# 5G Network Architecture
+
+5G was built with network functions divided by services which is known as the 5G core Service-Based Architecture (SBA). The 5G core is a network of interconnected services, as illustrated in the figure below.
+
+```{image} ../images/5g_network_architecture.png
+:alt: 5G Network Architecture
+:height: 600
+:align: center
+```
+
+The 5G Core Network is composed of various network functions, each serving a unique purpose. These functions communicate internally and externally over well-defined standard interfaces.
+The User Plane Function is a critical component of the 5G core network architecture It oversees the management of user data during the data transmission process. The UPF serves as a connection point between the RAN and the data network.
+UPF Interfaces/reference points with employed protocols:
+- N3 (GTP-U): Interface between the RAN (gNB) and the UPF
+- N9 (GTP-U): Interface between two UPF’s
+- N6 (GTP-U): Interface between the Data Network (DN) and the UPF
+- N4 (PFCP): Interface between the Session Management Function (SMF) and the UPF  
```

Comment:
I think we want to focus on the actual network interfaces between the UPF and AMF (and potentially SMF). In any case, @ghislainbourgeois and @markbeierl would be better placed to review this here.

Reply:
OK, I renamed the interfaces according to our scenario.

Reply:
Done